content/docs/hello-world.md

```diff
@@ -6,7 +6,7 @@ prev: cdn-links.html
 next: introducing-jsx.html
 ---
 
-The smallest React example looks like this:
+Ett litet React exempel ser ut så här:
```

Comment:
The first part of the compound noun "React exempel" is a proper noun and thus it should probably be together with a hyphen as such "React-exempel". Refer to this page for some more details https://resurs.slu.se/samskriva-eller-sarskriva-one-word-or-two/